src/neo/SmartContract/Native/Tokens/NeoToken.cs

```diff
@@ -89,6 +89,8 @@ private BigInteger CalculateBonus(StoreView snapshot, BigInteger value, uint sta
             return value * sum * NeoHolderRewardRatio / 100 / TotalAmount;
         }
 
+        private bool shouldRefreshCommittee(uint height) => height % (ProtocolSettings.Default.CommitteeMembersCount + ProtocolSettings.Default.ValidatorsCount) == 0;
```

Comment:
The first letter of methods must be capitalized.

Reply:
Have fixed